mk/gx.mk

```diff
@@ -1,7 +1,11 @@
-gx-path = gx/ipfs/$(shell gx deps find $(1))/$(1)
+binpostfix :=
+ifeq ($(OS),Windows_NT)
+binpostfix = .exe
+endif
+gx-path = gx/ipfs/$(shell gx$(binpostfix) deps find $(1))/$(1)
```

Comment:
`gx` in this file should be avialable from PATH variable so path or the postfix should not be needed.

Reply:
Does it work without it?